.github/workflows/general.yml

```diff
@@ -65,12 +65,14 @@ jobs:
 
       - name: Run cargo fmt
         uses: actions-rs/cargo@v1
+        continue-on-error: true  # WARNING: only for this example, remove it!
         with:
           command: fmt
           args: --all -- --check
 
       - name: Run cargo clippy
         uses: actions-rs/cargo@v1
+        continue-on-error: true  # WARNING: only for this example, remove it!
```

Comment:
Maybe we can add `#[allow(clippy::$LINT_NAME)]` in the source for now with a FIXME but keep erroring in CI?